biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -1395,7 +1398,19 @@ public Container getBundle(String bsn, String range, Strategy strategy, Map<Stri
 	 * @return a list of relevant repositories used by various methods.
 	 */
 	public List<RepositoryPlugin> getRepositories() {
-		return workspace.getRepositories();
+		// if (tags == null || tags.includesAny(Constants.REPOTAGS_RESOLVE)) {
+
+		return workspace.getRepositories()
+			.stream()
+			.filter(repo -> {
+				Tags tags = repo.getTags();
+
+				if (tags == null || tags.includesAny(Constants.REPOTAGS_RESOLVE)) {
```

Comment:
Should it not be NO_RESOLVE ?

Reply:
I think it is correct as it is. My thinking was to use the same logic which is used for Resolving 

https://github.com/bndtools/bnd/commit/2688cb56f84db9e1619a7af6a4c2316fb39cafd6#diff-096900d1f971d5ff7606d028e316560c8e89c25031bb473a493b776642f7f347R34 called by 

https://github.com/chrisrueger/bnd/blob/180a01a327d1ca3b001206ace8d024dd3994cd7a/biz.aQute.resolve/src/biz/aQute/resolve/BndrunResolveContext.java#L410

In other words: The repositories a Project should "care" about are the same repositories which are considered for resolving by `BndrunResolveContext.java`